packages/openapi-to-graphql/README.md

```diff
@@ -164,6 +164,8 @@ The options object can contain the following properties:
 
 - `customResolvers` (type: `object`, default: `{}`): OpenAPI-to-GraphQL, per default, creates resolver functions that make REST calls to resolve fields in the generated GraphQL interface. This option allows users to provide custom resolver functions to be used in place of said ones created by OpenAPI-to-GraphQL. The field that the custom resolver will affect is identifed first by the [title](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#infoObject) of the OAS, then the [path](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#paths-object) of the operation, and lastly the [method](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#path-item-object) of the operation. The `customResolvers` object is thus a triply nested object where the outer key is the title, followed by the path, and finally the method, which points to the [resolver function](https://graphql.org/learn/execution/#root-fields-resolvers) itself. The resolver function can use the parameters `obj`, `args`, `context`, and `info` in order to produce the proper data, as do standard [resolver functions](https://graphql.org/learn/execution/#root-fields-resolvers) in GraphQL. Use cases include the resolution of complex relationships between types, implementing performance improvements like caching, or dealing with non-standard authentication requirements. _Note: Because the arguments are provided by the GraphQL interface, they may look different from the [parameters](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#parameterObject) defined by the OAS. For example, they will have [sanitized](https://github.com/Alan-Cha/openapi-to-graphql#characteristics) names. The [request body](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#requestBodyObject) will also be contained in the arguments as an [input object type](https://graphql.org/graphql-js/mutations-and-input-types/)._
 
+- `addSlicingPattern` (type: `boolean`, default: `false`): [Pagination](https://graphql.org/learn/pagination/) allows for control over fields that return lists of objects. There are different pagination models such as the slicing pattern, which adds arguments to fields, and the connections pattern, which in addition to new arguments also introduces a number of new types and offers finer control. This option will follow the former model, adding `first` and `last` arguments to fields that return lists of objects.
```

Comment:
I am not sure whether we even want to mention the connections pattern here.

Reply:
Connections pattern is something I want to look into so I thought I might as well mention it here now. 

On the topic of connections pattern, I feel like I should mention implementing the connections pattern will be quite a daunting task. First of all, it will probably have a huge affect on how we handle resolvers and link resolvers. Secondly, it raises a bunch of open questions: how do we implement a `cursor`? I think we need knowledge about the object’s `id`. We can pass the responsibility to the user but they would need to take care of it through the `customResolvers` option but the way it is currently set up (i.e. using the path and the method) does not make it easy to manipulate the resolvers kf the `edges` and `nodes` types.

Reply:
I fully agree. Implementing the connections pattern will in many cases be impossible, IMO. Because it depends on certain features (slicing arguments with `first` and `last` semantic, `cursors` etc.) to be exposed by the underlying API. I am not at all confident we will be able to support this for arbitrary APIs in OASGraph.